CONTRIBUTING.md

```diff
@@ -0,0 +1,305 @@
+framework: tf
+license: https://raw.githubusercontent.com/pudae/tensorflow-densenet/master/LICENSE
+```
+----
+*After this step you will obtain **model.yml** file*
+
+## Model conversion
+
+Deep Learning Inference Engine (IE) supports models in Intermediate Representation (IR) format. A model from any supported framework can be converted to IR using Model Optimizer tool included in OpenVINO&trade; toolkit. Find more information about conversion in [Model Optimizer Developer Guide](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html). After successful conversion you will get model in IR format `*.xml` representing net graph and `*.bin` containing net parameters. 
+
+> **NOTE 1**: image pre-processing parameters (mean and scale) should be built into converted model to simplify model usage.
+
+> **NOTE 2**: if model input is a color image, color channel order should be `BGR`.
+
+*After this step you`ll get **conversion parameters** for Model Optimizer.*
+
+## Demo
+
+The demo shows the main idea of model inference using IE. If your model solves one of the tasks supported by Open Model Zoo, find appropriate demo from [demos](https://docs.openvinotoolkit.org/latest/_demos_README.html) or sample from [samples](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_Samples_Overview.html).
+
+If appropriate demo or sample are absent, you must provide your own demo (C++ or Python). Demos are required to support the following keys:
+
+-    `-i "<input>"`             Required. Input to process.
+-    `-m "<path>"`              Required. Path to an .xml file with a trained model.
+-    `-d "<device>"`            Optional. Target device for model inference. Default is CPU.
+-    `-no_show`                 Optional. Do not visualize inference results.
+
+> Note: For Python is preferable to use `-` instead of `_` as word separators (e.g. `-no-show`)
+
+Also you can add any other necessary parameters.
+
+If you adding new demo, please provide auto-testing support too:
+- add demo launch parameters in [demos/tests/cases.py](demos/tests/cases.py)
+- prepare list of input images in [demos/tests/image_sequences.py](demos/tests/image_sequences.py)
+
+*After this step you'll get **demo** for your model (if no demo was available)*
+
+## Accuracy validation
+
+Accuracy validation can be performed by [Accuracy Checker](./tools/accuracy_checker) tool, provided with repository. This tool can use IE to run converted model or original framework to run original model. Accuracy Checker supports lot of datasets, metrics and preprocessing options, what makes validation quite simple (if task is supported by tool). You need only create configuration file, which contain necessary parameters to do accuracy validation (specify dataset and annotation, pre- and post processing parameters, accuracy metric to compute and so on). More details you can find [here](./tools/accuracy_checker#testing-new-models).
```

Comment:
> Accuracy Checker supports lot of datasets

`a ` lot of OR lot`s` of.
By the way this does not clarify anything. Should there be a link to lists of datasets of something like that?

> pre- and post

use both with `-` or none.

`More details you can find [here](./tools/accuracy_checker#testing-new-models).` -> `You can find (or just Find) more details [here](./tools/accuracy_checker#testing-new-models).`